### PR TITLE
[DOC] Rewrite Fluid introduction chapter

### PR DESCRIPTION
**Pull request by s2b**
This is an attempt to streamline and to reduce the introduction to Fluid. I tried to reduce the dependencies, such as language handling, which should probably be part of a separate chapter. I also tried to make it more practical by providing some concrete examples.

I'm not sure if the actual implementation of partials and layouts should already be explained here or if it's sufficient to explain the concept and pick up the implementation at a later stage in the tutorial.

### CODE REVIEW

File: Documentation/Concepts/Fluid/Index.rst [hunk 1]
```diff
@@ -7,159 +7,178 @@
...
+Using Control Structures
+------------------------
+
+ViewHelpers are also used for so-called control structures.
+If you want to add a condition to your template, you can
+use the `<f:if>` ViewHelper:
+
+..  code-block:: xml
+
+    <f:if condition="{myVariable} == 'hello'">
```
Comment (garvinhicking) [inline Documentation/Concepts/Fluid/Index.rst:87]: Could you add info about compound conditions and else-if too, this is a common "problem" to actually use conditions.
(Or link to f:if viewhelper docs?)

File: Documentation/Concepts/Fluid/Index.rst [hunk 2]
```diff
@@ -7,159 +7,178 @@
...
+        The variable is "hello".
+    </f:if>
+
+You can also use the `<f:for>` ViewHelper to loop over an
```
Comment (garvinhicking) [inline Documentation/Concepts/Fluid/Index.rst:91]: Link to the f:for VH?

File: Documentation/Concepts/Fluid/Index.rst [hunk 3]
```diff
@@ -7,159 +7,178 @@
 Fluid templates
 ===============
 
-..  todo: This chapter was moved from the site package tutorial:
-    Overhaul and improve
-
 ..  _quick-introduction-to-fluid:
```
Comment (Alagts) [inline Documentation/Concepts/Fluid/Index.rst:10]:
```suggestion
..  contents::
    :caption: Content on this page
    
..  _quick-introduction-to-fluid:
```

**Review by Alagts — COMMENTED** [review]
I would suggest to add a content overview on each page. This enhances readability and accessibility.

**Review by Alagts — APPROVED** [review]

**Pull request merged**